src/modules/vendor/components/programs/cards/CardContainer.vue

```diff
@@ -3,19 +3,22 @@
     <q-scroll-area ref="cardContainer" :thumb-style="{ width: '6px', 'border-radius': '10px' }"
       :content-style="{ display:'flex', 'flex-direction': 'column' }"
       :content-active-style="{ display:'flex', 'flex-direction': 'column' }">
-      <div v-for="(card, index) in cards" :key="index" :class="getCardStyle(card)">
-        <div class="card-actions">
-          <ni-button v-if="isSelected(card)" icon="delete" @click.native="deleteCard(card)" :disable="disableEdition" />
-        </div>
-        <div class="card-cell cursor-pointer" @click="selectCard(card)">
-          <div class="card-cell-title">
-            <div class="text-weight-bold">{{ index + 1 }}. {{ getHeading(card) }}</div>
-            <q-icon v-if="disableEdition" name="lock" size="xs" :class="{'locked-unselected': !isSelected(card)}" />
+      <draggable v-model="draggableCards" @end="dropCard()" ghost-class="ghost" :disabled="$q.platform.is.mobile">
+        <div v-for="(card, index) in cards" :key="index" :class="getCardStyle(card)">
```

Comment:
il y a un truc bizarre au niveua de l'affichage donc la je epnse qu'il faut que tu boucles dans ton v-for sur `draggableCards`

Reply:
tadam